modules/ROOT/pages/clauses/delete.adoc

```diff
@@ -53,6 +54,28 @@ This query is only possible to run on nodes without any relationships connected
 Deleted 1 node
 ----
 
+=== NODETACH keyword
+
+_This feature was introduced in Neo4j 5.14._
+
+It is also possible to delete the single node using the `NODETACH DELETE` clause.
+This behaves the same as omitting it.
+
+.Query
+[source, cypher, indent=0]
```

Comment:
We don't need the indent=0 anymore (I have been meaning to get rid of all existing examples but havent yet)

```suggestion
[source, cypher]
```